app/views/course/experience_points_disbursement/new.slim

```diff
@@ -10,6 +10,24 @@
   = f.input :reason, label: false
 
   h3 = t('.awardees')
+  
+  - if current_course.groups.size >= 1
+    div.dropdown
+      button.btn.btn-default.dropdown-toggle data-toggle='dropdown'
+        - current_group = @experience_points_disbursement.group
+        => current_group.present? ? t('.filter_selected', group: format_inline_text(current_group.name)) \
```

Comment:
No need `present?`, it's normally used for checking string and arrays. 

`current_group ? t('.filter_selected', group: format_inline_text(current_group.name)) : t('.filter_not_selected')` will do.